frontend/encore/versioning.rst

```diff
@@ -66,7 +66,7 @@ like normal:
 
 .. code-block:: html+twig
 
-    <img src="{{ asset('build/images/logo.png') }}">
+    <img src="{{ asset('build/images/logo.png') }}" alt="my great image">
```

Comment:
Good catch! But given that the example is about a logo, we should use something like `alt="ACME logo"`

Reply:
@javiereguiluz done 😉